docs/source/overview.rst

```diff
@@ -61,23 +61,29 @@ please cite :cite:`pfister-2021-scico`
 in the source distribution).
 
 
-Anatomy of an Inverse Problem
------------------------------
+Inverse Problems
+----------------
+
+SCICO is designed to solve inverse problems such as
+
+.. math::
+
+   \argmin_{\mb{x}} \; f(\mb{x}) + g(\mb{x})
 
-SCICO is designed to solve problems of the form
+or
 
 .. math::
 
-   \argmin_{\mb{x}} \; f(\mb{x}) + \sum_{k=1}^K g_k(C_k (\mb{x})),
+   \argmin_{\mb{x}} \; f(\mb{x}) + g(C \mb{x}) \;,
```

Comment:
Remove the first form (f(x) + g(x)) and just show this one? It seems odd to explicitly mention two very similar problems.

Reply:
Good point. Done.